survey_form/index.html

```diff
@@ -43,8 +43,94 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form-survey__title">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label class="form-survey__label" for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
+
+      <label class="form-survey__label" for="email">Email: </label>
+      <input
+        type="email"
+        id="email"
+        class="form-survey__input"
+        required
+        placeholder="leonardo.dicaprio@gmail.com"
+      />
+
+      <label class="form-survey__label" for="number">Age: </label>
```

Comment:
Wcześniej nie zauważyłem, ale jeszcze `for` można zmienić na `age`, to samo będzie z `id` inputa odpowiadającego temu labelowi. Będzie bardziej semantycznie i będzie zgodnie z konwencją `name` i `email`. Ogólnie według mnie konwencje są ważne, szczególnie w większych projektach :)

Reply:
na początku dałem age, ale zmieniłem tak jak pisali na stronie freeCodeCampu :)

Reply:
Wtedy można się zastanawiać, co się powinno wydarzyć, gdy mamy dwa inputy numeryczne ;)